CRM/Core/ClassLoader.php

```diff
@@ -104,7 +104,8 @@ public function register($prepend = FALSE) {
     }
     $civicrm_base_path = dirname(dirname(__DIR__));
 
-    require_once dirname(dirname(__DIR__)) . '/vendor/autoload.php';
+    // @todo: need some way to check if we should register vendor or not
```

Comment:
Sure, we could do `class_exists()` on a class from a CiviCRM dependency. I think I'd prefer something more direct, though, like checking if the vendor directory exists, which it shouldn't if installed via composer as opposed to from a tarball... Lemme try that!